index.js

```diff
@@ -146,10 +146,21 @@ class PluginXrpPaychan extends PluginBtp {
   }
 
   async _handleData (from, { requestId, data }) {
-    if (!this._paychanReady) throw new Error('paychan initialization has not completed or has failed.')
```

Comment:
This seems dangerous; the plugin is in an inconsistent state so there might be some unintended consequences or vulnerabilities